CHANGELOG.md

```diff
@@ -2,13 +2,177 @@
 
 ## 10.0.0
 
-Teleport 10.0 is a major release of Teleport that contains new features, improvements, and bug fixes.
+Teleport 10 is a major release that brings:
+
```

Comment:
Small thing, but should we put "Preview" in parentheses in these bullets/headings, e.g., "Passwordless (Preview)", to distinguish feature names from release statuses?

Reply:
Good call, I think we should.